lib/internal/modules/helpers.js

```diff
@@ -342,8 +342,22 @@ let _hasStartedUserCJSExecution = false;
 // there is little value checking whether any user JS code is run anyway.
 let _hasStartedUserESMExecution = false;
 
+/**
+ * Load a public builtin.
+ * @param {string} id id of the builtin to be loaded. ID must be without the node: prefix.
+ * @returns {object} exports of the builtin.
+ */
+function getBuiltin(id) {
+  validateString(id, 'id');
+  if (!BuiltinModule.canBeRequiredByUsers(id)) {
+    throw new ERR_UNKNOWN_BUILTIN_MODULE(id);
+  }
+  return require(id);
```

Comment:
```suggestion
  return require(`node:${id}`);
```
this seems more safe to me

Reply:
unless we support passing ids that include the `node:` prefix

Reply:
Because internal modules are CJS modules, the `require` function that is available to them is not the same as the one available in userland CJS modules – one difference is that it doesn’t support `node:` prefix, another one is that it can only load built-in modules (including the ones in `lib/internal/`).

Reply:
This is the internal require, it doesn’t recognize the node prefix.